pl-l2d42c8/pl-l2d42c8.xml

```diff
@@ -59,15 +59,15 @@
           est, quod interdum ex superbia non proveniat; nullum etiam quod ex cupiditate
           aliquando non descendat. Sunt enim nonnulli hominum, qui ex cupiditate fiunt superbi; et
           aliqui ex superbia fiunt cupidi. Est enim, ut ait<quote>
-
+        <ref>
           <name>Augustinus<name>
+        <ref>
 
-            <quote>
-          , homo, qui non esset
-          amator pecuniae, nisi per hoc putaret, se excellentiorem esse, ideoque, ut excellat,
-          divitias cupit; tali ex superbia oboritur cupiditas. Et est aliquis, qui non amaret
-          excellere nisi putaret, per hoc maiores divitias habere.
-            <quote>
+        <quote>
+          , homo, qui non esset amator pecuniae, nisi per hoc putaret, se excellentiorem esse, 
```

Comment:
the comma before "homo" is not really part of the quote. So this should really look like the following: 

```
<ref>Augustinus</ref>,
<cit>
   <quote>
     text text
   </quote>
</cit>
```